docs/rpc-spec.md

```diff
@@ -315,6 +315,8 @@ Files are returned in the order they are laid out in the torrent. References to
 | Key | Value Type | transmission.h source
 |:--|:--|:--
 | `address`            | string     | tr_peer_stat
+| `bytesToClient`      | number     | tr_peer_stat
+| `bytesToPeer`        | number     | tr_peer_stat
```

Comment:
Please change the case style to snake_case. All new API should use snake_case from `4.1.0`. (See #2855)